wasm/src/record/record_plaintext.rs

```diff
@@ -92,6 +105,27 @@ mod tests {
         assert_eq!(record.gates(), 99);
     }
 
+    #[wasm_bindgen_test]
+    fn test_serial_number() {
+        let pk = PrivateKey::from_string("APrivateKey1zkpDeRpuKmEtLNPdv57aFruPepeH1aGvTkEjBo8bqTzNUhE").unwrap();
```

Comment:
Unwraps however are fine in tests, and actually "mostly" desirable because you want the tests to fail if you're expecting success (that is, unless you're testing for specific Error variants - but that's a convo for another time)